src/stories/Blocks/footer/footer.scss

```diff
@@ -47,92 +35,33 @@
   }
 }
 
-// Widgets
-.footer__widgets {
-  display: flex;
-  justify-content: space-between;
+.footer-title {
+  @include typography($typo__h4);
+  @include media-query__small {
+    margin-bottom: $s-md;
+  }
 }
 
-// Dropdown tweaks
-.footer .dropdown {
-  width: 82px;
+.footer-content {
+  @include typography($typo__body-placeholder);
 
-  .dropdown__select {
-    background-color: $color__global-secondary;
-    height: $s-xl;
-    padding: 0 12px;
-    min-width: 82px;
+  a {
+    display: block;
+    text-decoration: none;
   }
 
-  .dropdown__arrows {
-    height: $s-xl;
-    width: $s-xl;
+  p {
+    margin-bottom: $s-md;
   }
-}
 
-// Site info
-.footer__desktop .footer__site-info {
-  display: flex;
-  justify-content: space-between;
-}
-
-.footer__site-info-links {
-  display: flex;
-
-  li {
-    padding-bottom: $s-md;
+  &[hidden] {
+    display: none;
```

Comment:
`display: none` will hide this element completely from screen readers. is that what you want? :)